SUPPORTEDTECH.rst

```diff
@@ -0,0 +1,111 @@
+    +--------------+------------------------+-----------------------------------------+-----------------------+
+    | FSS          | Semi-honest            | xor                                     |                    2+ |
+    +--------------+------------------------+-----------------------------------------+-----------------------+
+    | FSS          | Semi-honest            | Equal Comparison                        |                    2+ |
+    +--------------+------------------------+-----------------------------------------+-----------------------+
+    | FSS          | Semi-honest            | Non Equal Comparison                    |                    2+ |
+    +--------------+------------------------+-----------------------------------------+-----------------------+
+    | FSS          | Semi-honest            | Less or Equal Comparison                |                    2+ |
+    +--------------+------------------------+-----------------------------------------+-----------------------+
+    | FSS          | Semi-honest            | Greater or Equal Comparison             |                    2+ |
+    +--------------+------------------------+-----------------------------------------+-----------------------+
+    | FSS          | Semi-honest            | Less than Comparison                    |                    2+ |
+    +--------------+------------------------+-----------------------------------------+-----------------------+
+    | FSS          | Semi-honest            | Greater than Comparison                 |                    2+ |
+    +--------------+------------------------+-----------------------------------------+-----------------------+
+    | FSS          | Semi-honest            | Mult, matmul, conv2d, conv2d\_transpose |                     2 |
+    +--------------+------------------------+-----------------------------------------+-----------------------+
+    | SPDZ         | Semi-honest            | Addition                                |                    2+ |
+    +--------------+------------------------+-----------------------------------------+-----------------------+
+    | SPDZ         | Semi-honest            | Substraction                            |                    2+ |
+    +--------------+------------------------+-----------------------------------------+-----------------------+
+    | SPDZ         | Semi-honest            | Division                                |                    2+ |
+    +--------------+------------------------+-----------------------------------------+-----------------------+
+    | SPDZ         | Semi-honest            | xor                                     |                    2+ |
+    +--------------+------------------------+-----------------------------------------+-----------------------+
+    | SPDZ         | Semi-honest            | Equal Comparison                        |                    2+ |
```

Comment:
It doesn't have to "Less than Comparison","Greater than Comparison" or "Equal Comparison". Just "Less than", "Greater Than", "Equal to", etc will do.